include/nccl_ofi_tuner.h

```diff
@@ -11,7 +11,7 @@
  * can come later (once a proto+algo combination is chosen, we can compute the
  * cost with different channel count and optimize for it.
  */
-OFI_NCCL_PARAM_INT(tuner_num_channels, "TUNER_NUM_CHANNELS", 8);
+OFI_NCCL_PARAM_INT(tuner_num_channels, "TUNER_NUM_CHANNELS", 81);
```

Comment:
?

Reply:
This value is divided by 10.0 elsewhere.